docs/data-modeling/projections.md

```diff
@@ -0,0 +1,540 @@
+3. │     233.25 │ 1211709305 │                18 │
+   └────────────┴────────────┴───────────────────┘
+
+3 rows in set. Elapsed: 0.005 sec.
+--highlight-next-line
+Processed 10.97 thousand rows,
+175.55 KB (2.20 million rows/s., 35.13 MB/s.)
+Peak memory usage: 58.08 KiB.
+```
+
+Notice how we were able to decrease the query time from 0.009 seconds to 0.005
+seconds and we scanned only 10.97 thousand rows instead of 3 million.
+
+We can confirm that our query above did indeed use the projection we made by
+querying the `system.query_log` table:
+
+```sql
+SELECT query, projections 
+FROM system.query_log 
+WHERE query_id='<query_id>'
+```
+
+```response
+   ┌─query─────────────────────────────────────────────────────────────────────────┬─projections──────────────────────┐
+1. │ SELECT                                                                       ↴│ ['default.trips.prj_tip_amount'] │
+   │↳  tip_amount,                                                                ↴│                                  │
+   │↳  trip_id,                                                                   ↴│                                  │
+   │↳  dateDiff('minutes', pickup_datetime, dropoff_datetime) AS trip_duration_min↴│                                  │
+   │↳FROM trips WHERE tip_amount > 200 AND trip_duration_min > 0                   │                                  │
+   └───────────────────────────────────────────────────────────────────────────────┴──────────────────────────────────┘
+```
+
+### Using projections to speed up UK price paid queries {#using-projections-to-speed-up-UK-price-paid}
+
+To demonstrate how Projections can be used to speed up query performance, let's
+take a look at an example using a real life dataset. For this example we'll be 
+using the table from our [UK Property Price Paid](https://clickhouse.com/docs/getting-started/example-datasets/uk-price-paid)
+tutorial with 30.03 million rows. This dataset is also available within our 
+[sql.clickhouse.com](https://sql.clickhouse.com/?query_id=6IDMHK3OMR1C97J6M9EUQS)
+environment.
+
+If you've not done so already, run the queries below to create the table and 
+insert the data.
+
+<details>
+<summary>Create and insert data</summary>
+```sql title="Create Table"
+CREATE TABLE uk_price_paid
+(
+    price UInt32,
+    date Date,
+    postcode1 LowCardinality(String),
+    postcode2 LowCardinality(String),
+    type Enum8('terraced' = 1, 'semi-detached' = 2, 'detached' = 3, 'flat' = 4, 'other' = 0),
+    is_new UInt8,
+    duration Enum8('freehold' = 1, 'leasehold' = 2, 'unknown' = 0),
+    addr1 String,
+    addr2 String,
+    street LowCardinality(String),
+    locality LowCardinality(String),
+    town LowCardinality(String),
+    district LowCardinality(String),
+    county LowCardinality(String)
+)
+ENGINE = MergeTree
+ORDER BY (postcode1, postcode2, addr1, addr2);
+```
+
+```sql title="Preprocess and insert the data"
+INSERT INTO uk_price_paid
```

Comment:
drop its in sql.clickhouse.com - add a note to look at the datasets on how to load the data